File: FC7/setupfc7.cpp
#include "setupfc7.h"
#include "ui_setupfc7.h"

setupFC7::setupFC7(QWidget *parent)
    : QMainWindow(parent)
    , ui(new Ui::setupFC7)
{
    ui->setupUi(this);
}

setupFC7::~setupFC7()
{
    delete ui;
}

File: widgets/SetupFC7.py
from PyQt5.QtWidgets import *
from PyQt5.QtCore import *
from PyQt5.QtGui import *
from PyQt5.uic import *

class SetupFC7(QWidget):
    def __init__(self, parent):
        super().__init__(parent)
        ui = loadUi('widgets_ui/setup_FC7.ui', self) 